sdk/storage/azure-storage-queue/azure/storage/queue/queue_service_client.py

```diff
@@ -0,0 +1,426 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import functools
+from typing import (  # pylint: disable=unused-import
+    Union, Optional, Any, Iterable, Dict, List,
+    TYPE_CHECKING)
+try:
+    from urllib.parse import urlparse
+except ImportError:
+    from urlparse import urlparse # type: ignore
+
+from ._shared.shared_access_signature import SharedAccessSignature
+from ._shared.models import LocationMode, Services
+from ._shared.utils import (
+    StorageAccountHostsMixin,
+    parse_query,
+    parse_connection_str,
+    process_storage_error)
+from ._generated import AzureQueueStorage
+from ._generated.models import StorageServiceProperties, StorageErrorException
+
+from .models import QueuePropertiesPaged
+from .queue_client import QueueClient
+
+if TYPE_CHECKING:
+    from datetime import datetime
+    from azure.core import Configuration
+    from azure.core.pipeline.policies import HTTPPolicy
+    from ._shared.models import AccountPermissions, ResourceTypes
+    from .models import (
+        QueueProperties,
+        Logging,
+        Metrics,
+        CorsRule
+    )
+
+
+class QueueServiceClient(StorageAccountHostsMixin):
+    """A client interact with the Queue Service at the account level.
+
+    This client provides operations to retrieve and configure the account properties
+    as well as list, create and delete queues within the account.
+    For operations relating to a specific queue, a client for this entity
+    can be retrieved using the :func:`~get_queue_client` function.
+
+    :param str account_url:
+        The URL to the queue storage account. Any other entities included
```

Comment:
minor: queue storage endpoint. storage account includes all 3 services.

Reply:
Fixed in PR: https://github.com/Azure/azure-sdk-for-python/pull/6186